### DISCUSSION TITLE
Batch search documentation

### DISCUSSION THREAD

**Discussion by mausch** [osm-search/Nominatim#2643] [Q&A] [answered]
Hi, I was just looking for a way to do batch geocoding and found https://github.com/osm-search/Nominatim/commit/a974d2509257cb2c4967631cea30eb483c37c837 , but I can't seem to find any documentation about this?

Is this still supported? deprecated? Maybe it's just missing in the docs?

Many thanks

**Comment by mtmail** [comment, marked as answer]
Undocumented and potentially broken https://nominatim.org/release-docs/latest/customize/Settings/#nominatim_search_batch_mode
